python/test/test_comb.py

```diff
@@ -120,7 +133,7 @@ def test_make_pureDP_to_zCDP():
         dp.m.make_gaussian(*input_space, 10.)
     ])
 
-    print(meas.map(1.))
+    assert meas.map(1.) == 0.010000000000000002
```

Comment:
Maybe round to make this less fragile? Unless it really should be exactly this?

Reply:
It works out to exactly this every time, due to the float representation of .01 being greater than .01.